lib/Email/Stuffer/TestLinks.pm

```diff
@@ -0,0 +1,77 @@
+package Email::Stuffer::TestLinks;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.010';
+
+use Test::Most;
+use Mojo::UserAgent;
+use Email::Stuffer;
+use Class::Method::Modifiers qw/ install_modifier /;
+
+=head1 SYNOPSIS
+
+    use Email::Stuffer::TestLinks;
+
+=head1 NAME
+
+Email::Stuffer::TestLinks - validates links in HTML emails sent by Email::Stuffer>send_or_die()
+
+=head1 DESCRIPTION
+
+When this module is included in a test, it parses HTML links (<a href="xyz"...) in every email sent through Email::Stuffer->send_or_die(). Each URI must get a successful response code (200 range) and the returned pagetitle must not contain 'error' or 'not found'.
+
+=cut
+
+install_modifier 'Email::Stuffer', after => send_or_die => sub {
+
+    my $self = shift;
+    my $ua   = Mojo::UserAgent->new;
+    $ua->max_redirects(10);
+    $ua->connect_timeout(5);
```

Comment:
I suspect you could also just

```perl
my $ua = Mojo::UserAgent->new(max_redirects => 10, connect_timeout => 5);
```

here.  There is also an alternative style that takes advantage of Mojo::UA isa Mojo::Base, so you can use a [K-combinator ](https://metacpan.org/pod/Mojo::Base#tap) to create and initialize the object at once:

```perl
my $ua = Mojo::UserAgent->new->tap(max_redirects => 10)->tap(connect_timeout => 5);
```

Reply:
You're right (I love cutting lines of code!)